pages/api/graphql-rest.page.ts

```diff
@@ -1363,32 +1383,21 @@ const cors = Cors({
 
 const apolloServer = new ApolloServer({
   schema,
-  dataSources: () => {
-    return {
-      mpdxRestApi: new MpdxRestApi(),
-    };
-  },
-  context: ({ req }: { req: NextApiRequest }): Partial<Context> => {
-    return { authHeader: req.headers.authorization };
-  },
+  allowBatchedHttpRequests: true,
 });
 
-const startServer = apolloServer.start();
-
-export default cors(async (req, res) => {
+export default cors((req, res) => {
   if (req.method === 'OPTIONS') {
     res.end();
     return false;
   }
-  await startServer;
-  await apolloServer.createHandler({
-    path: '/api/graphql-rest',
+  startServerAndCreateNextHandler(apolloServer, {
```

Comment:
This should resolve the `API resolved without sending a response for /api/graphql-rest, this may result in stalled requests.` errors I'm seeing.

```suggestion
export default cors(async (req, res) => {
  if (req.method === 'OPTIONS') {
    res.end();
    return false;
  }
  await startServerAndCreateNextHandler(apolloServer, {
```